rdt/__init__.py

```diff
@@ -33,26 +33,46 @@ def get_demo(dtypes=('int', 'float', 'str', 'datetime'), nans=0.2, size=10):
     Returns:
         pd.DataFrame
     """
-    if np.isscalar(nans):
-        nans = [nans] * len(dtypes)
-
-    columns = {}
-    for count, (dtype, nan) in enumerate(zip(dtypes, nans)):
-        if dtype == 'int':
-            column = np.random.randint(100, size=size)
-        elif dtype == 'float':
-            column = np.random.random(size) * 100
-        elif dtype == 'str':
-            column = np.random.choice(['a', 'b', 'c', 'd'], size=size)
-        elif dtype == 'datetime':
-            deltas = np.random.randint(1000000, size=10)
-            datetimes = np.array([np.datetime64('2019-10-13T18:34')] * size)
-            column = datetimes + deltas
-
-        column = pd.Series(column)
-        nan_index = np.random.choice(range(size), size=int(size * nan), replace=False)
-        column.iloc[nan_index] = np.nan
-
-        columns[f'{count}_{dtype}'] = column
-
-    return pd.DataFrame(columns)
+    # Hard code first five rows
+    login_dates = ['2021-06-26', '2021-02-10', 'NAT', '2020-09-26', '2020-12-22']
+    last_login = [np.datetime64(i) for i in login_dates]
+    email_optin = pd.Series([False, False, False, True, np.nan], dtype='object')
+    credit_card = ['VISA', 'VISA', 'AMEX', np.nan, 'DISCOVER']
+    age = [29, 18, 21, 45, 32]
+    dollars_spent = [99.99, np.nan, 2.50, 25.00, 19.99]
+
+    data = pd.DataFrame({
+        'last_login': last_login,
+        'email_optin': email_optin,
+        'credit_card': credit_card,
+        'age': age,
+        'dollars_spent': dollars_spent
+    })
+
+    if num_rows <= 5:
+        return data.iloc[:num_rows]
+
+    # Randomly generate the remaining rows
+    np.random.seed(42)
```

Comment:
Why do we use seed ?

Reply:
From the issue: "_Set a seed so we always return the same rows_"